backend/notifications/tests.py

```
@@ -0,0 +1,347 @@
+
+    def test_requires_authentication(self):
+        client = APIClient()
+        response = client.get('/api/v1/notifications/')
+        self.assertIn(response.status_code, (401, 403))
+
+
+class BroadcastAdminMixinTests(TestCase):
+    def setUp(self):
+        self.admin_user = User.objects.create_superuser(
+            email='admin@test.com',
+            password='adminpass123',
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)